tests/suites/test_suite_psa_crypto.data

```diff
@@ -6984,27 +6984,27 @@ depends_on:PSA_WANT_ALG_CTR:PSA_WANT_KEY_TYPE_AES
 generate_key:PSA_KEY_TYPE_AES:64:PSA_KEY_USAGE_EXPORT | PSA_KEY_USAGE_ENCRYPT | PSA_KEY_USAGE_DECRYPT:PSA_ALG_CTR:PSA_ERROR_INVALID_ARGUMENT:0
 
```

Comment:
In “PSA generate key: bad type (RSA public key)”, please change the key size from 512 to `PSA_VENDOR_RSA_GENERATE_MIN_KEY_BITS`, so that there's only one thing wrong in the test case data.